packages/tomls/src/omnibus/reya_cronos.toml

```diff
@@ -59,6 +60,8 @@ ordersGatewayProxyPackage = "reya-orders-gateway:1.0.1@proxy"
 ordersGatewayRouterPackage = "reya-orders-gateway:1.0.7@router"
 oracleAdaptersProxyPackage = "reya-oracle-adapters:1.0.1@proxy"
 oracleAdaptersRouterPackage = "reya-oracle-adapters:1.0.5@router"
+shareTokensProxyPackage = "reya-share-tokens:1.0.0@proxy"
```

Comment:
hmm, can you pls remind me what's meant by the share tokens proxy package? there're multiple proxies for each share token that each share a single router like with tokens (tho i see tokens has a proxy as well)

Reply:
this is the package, there are actually more proxies